src/sparsezoo/analyze_v2/model_analysis.py

```diff
@@ -146,20 +146,23 @@ def analyze(path: str, download_path: Optional[str] = None) -> "ModelAnalysis":
     :param path: .onnx path or stub
     """
     if path.endswith(".onnx"):
-        onnx_model = load_model(path)
+        onnx_model = load_model(path, load_external_data=False)
+        onnx_model_path = path
     elif is_stub(path):
         model = Model(path, download_path)
         onnx_model_path = model.onnx_model.path
-        onnx_model = onnx.load(onnx_model_path)
+        onnx_model = onnx.load(onnx_model_path, load_external_data=False)
     else:
         raise ValueError(f"{path} is not a valid argument")
 
-    model_graph = ONNXGraph(onnx_model)
-    node_shapes, _ = extract_node_shapes_and_dtypes(model_graph.model)
+    node_shapes, _ = extract_node_shapes_and_dtypes(onnx_model, onnx_model_path)
```

Comment:
is the session cleaned up at this point so we aren't loading two copies of the full model in at once?

Reply:
should be - is there a way to explicitly make sure the inference session is cleaned up? deleting the object or explicitly running garbage collection?

seems like memory is released after the inference function completes/the session goes out of scope